client/classic/upe/payment-processing.js

```diff
@@ -0,0 +1,310 @@
+import {
+	appendIsUsingDeferredIntentToForm,
+	appendPaymentMethodIdToForm,
+	getPaymentMethodTypes,
+	getStripeServerData,
+	getUpeSettings,
+	showErrorCheckout,
+} from '../../stripe-utils';
+
+const gatewayUPEComponents = {};
+
+const paymentMethodsConfig = getStripeServerData()?.paymentMethodsConfig;
+
+for ( const paymentMethodType in paymentMethodsConfig ) {
+	gatewayUPEComponents[ paymentMethodType ] = {
+		elements: null,
+		upeElement: null,
+	};
+}
+
+/**
+ * Initializes the appearance of the payment element by retrieving the UPE configuration
+ * from the API and saving the appearance if it doesn't exist. If the appearance already exists,
+ * it is simply returned.
+ *
+ * @return {Object} The appearance object for the UPE.
+ */
+function initializeAppearance() {
```

Comment:
Just noticed this was empty, will this be implemented in a separate PR?

Reply:
Thanks! I meant to do it in this PR 🙂 fixed in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2753/commits/66977492f0ed1ee986675d50f87b2297132610b3

Reply:
Thanks!